src/utils/src/types/change-request-types.ts

```diff
@@ -4,6 +4,8 @@
  */
 
 export interface ChangeRequest {
-  submitter: String;
-  type: String;
+  id: number;
```

Comment:
what's the reason for changing the data definition? generally data definition changes are more critical and have the potential to break things, so unless explicitly detailed in the issue, you'll want to avoid data definition changes.